docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -341,6 +340,48 @@ Replace all `<>` place holders with the real values that we got from all previou
 ```
 - Note - **extent** taken from step 2 - where bbox is calculated.
 
+### Leaflet (1.x)
+```javascript
+  import L from "leaflet";
+...
+...
+...
+    //urlTemplate example: https://maps/api/raster/v1/wmts/test-orthophoto/{TileMatrixSet}/{TileMatrix}{TileCol}/{TileRow}.jpeg
+    const parser = (urlTemplate) =>{
+      return wmtsUrl
+    .replace("{TileMatrixSet}", '<LAYER_TILE_MATRIX_SET_ID>')         // from Step_4          
+    .replace("{TileMatrix}", "{z}")
+    .replace("{TileRow}", "{y}")
+    .replace("{TileCol}", "{x}");
+    }
+
+    const bounds = L.latLngBounds([
+      [extent[1], extent[0]],                                         // from Step_2
+      [extent[3], extent[2]],                                         // from Step_2
+    ]);
+
+    const urlTemplate = '<LAYER_WMTS_URL>'                                    // from Step_3 or Step_4
+    const parsedUrl = parser(urlTemplate)                                     
+                                                                      //in case of queryParameter authentication:
+    const layer = L.tileLayer(parsedUrl + 'token=${RASTER_TOKEN}',{   // received RASTER auth token
+      id : '<desired_layer_identifier>',                              // from Step_3
+      bounds
+    })
+
+    map.addLayer(layer);   
```

Comment:
map is not configured for `L.CRS.EPSG4326`

Reply:
added the map configuration